jwql/website/apps/jwql/data_containers.py

```diff
@@ -164,8 +164,18 @@ def get_current_flagged_anomalies(rootname):
         A list of currently flagged anomalies for the given ``rootname``
         (e.g. ``['snowball', 'crosstalk']``)
     """
-
-    query = di.session.query(di.Anomaly).filter(di.Anomaly.rootname == rootname).order_by(di.Anomaly.flag_date.desc()).limit(1)
+    inst_lower=inst.lower()   #####Put into MIXED case as seen in constants.py!
```

Comment:
@tnking97 You could probably get away from having to use all of these conditional statements if you do something like:

```python
table_dict = {}
for instrument in JWST_INSTRUMENT_NAMES:
    table_dict[instrument.lower()] = getattr(di, '{}Anomaly'.format(instrument))

table = table_dict[inst.lower()]
query = di.session.query(table).filter(table.rootname == rootname).order_by(table.flag_date.desc()).limit(1)
```

Reply:
@bourque Thanks for the suggestion! I will update that.